### DISCUSSION TITLE
VRS for Docker

### DISCUSSION THREAD

**Discussion by 76rhodan** [vradarserver/vrs#98] [General]
Hi,

I just wanted to drop a quick note about "my" project [here](https://github.com/76rhodan/docker-virtualradarserver/tree/dev) - (or actually [here](https://github.com/sdr-enthusiasts/docker-virtualradarserver). I working on my fork and push changes to the SDR-Enthusiasts every now and then - which is, btw, a great and helpful community). 
It's basically a docker image of VRS for Linux (supported hardware: `amd64`, `arm64`, `armhf`, `i386`) which should make the process of setting up and running VRS on a Linux system pretty easy and straight forward. Plugins are preinstalled as well.
There are lots of enhancements by [rikgale](https://github.com/rikgale) included, especially his database (and updates for it), silhouettes, side views, layers and markers.
Also, the container comes with an autoupdate feature, which keeps everything up to date with the latest (community) additions.

Currently, everything is still very much "work in progress" -  but in a running condition - and we're close to "MK I" - a release of a stable image of the latest VRS Preview 3 version. The latest additions are usually in `:dev`, `:main` is declared to be in stable condition.

Any questions / suggestions - let me know.

Have a good one!

**Comment by vradarserver** [comment]
Ha, very cool :)
